Shared/SongReader/Song.swift

```diff
@@ -27,11 +20,11 @@ struct Song: Hashable {
     private(set) var bookmark: Data? = nil
     
     init(title: String,
-              artist: String,
-              lyrics: String? = nil,
-              album: String? = nil,
-              cover: UIImage? = nil,
-              bookmark: Data? = nil)
+         artist: String,
+         lyrics: String? = nil,
+         album: String? = nil,
+         cover: UIImage? = nil,
+         bookmark: Data? = nil)
```

Comment:
Do you really need to specify nil here? It's an optional value so wouldn't always be initialized as nil?

Reply:
It's to synthesise a series of initialisers, although I did get some cleanup idea here, thanks man!

Reply:
These are parameters so require `= nil` to allow omission

Reply:
Derp. Missed the parameter part here. Makes sense. Thanks!